src/main/cli/settings.c

```diff
@@ -773,7 +774,18 @@ const clivalue_t valueTable[] = {
 #ifdef USE_BLACKBOX
     { "blackbox_p_ratio",           VAR_UINT16 | MASTER_VALUE, .config.minmaxUnsigned = { 0, INT16_MAX }, PG_BLACKBOX_CONFIG, offsetof(blackboxConfig_t, p_ratio) },
     { "blackbox_device",            VAR_UINT8  | HARDWARE_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_BLACKBOX_DEVICE }, PG_BLACKBOX_CONFIG, offsetof(blackboxConfig_t, device) },
-    { "blackbox_record_acc",        VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON }, PG_BLACKBOX_CONFIG, offsetof(blackboxConfig_t, record_acc) },
+    { "bb_log_pids",                VAR_UINT16 | MASTER_VALUE | MODE_BITSET, .config.bitpos = SELECT_FIELD_PID,   PG_BLACKBOX_CONFIG, offsetof(blackboxConfig_t, fields) },
+    { "bb_log_rc",                  VAR_UINT16 | MASTER_VALUE | MODE_BITSET, .config.bitpos = SELECT_FIELD_RC_COMMANDS,   PG_BLACKBOX_CONFIG, offsetof(blackboxConfig_t, fields) },
+    { "bb_log_setpoint",            VAR_UINT16 | MASTER_VALUE | MODE_BITSET, .config.bitpos = SELECT_FIELD_SETPOINT,   PG_BLACKBOX_CONFIG, offsetof(blackboxConfig_t, fields) },
+    { "bb_log_bat",                 VAR_UINT16 | MASTER_VALUE | MODE_BITSET, .config.bitpos = SELECT_FIELD_BATTERY,   PG_BLACKBOX_CONFIG, offsetof(blackboxConfig_t, fields) },
+    { "bb_log_mag",                 VAR_UINT16 | MASTER_VALUE | MODE_BITSET, .config.bitpos = SELECT_FIELD_MAG,   PG_BLACKBOX_CONFIG, offsetof(blackboxConfig_t, fields) },
+    { "bb_log_alt",                 VAR_UINT16 | MASTER_VALUE | MODE_BITSET, .config.bitpos = SELECT_FIELD_ALTITUDE,   PG_BLACKBOX_CONFIG, offsetof(blackboxConfig_t, fields) },
+    { "bb_log_rssi",                VAR_UINT16 | MASTER_VALUE | MODE_BITSET, .config.bitpos = SELECT_FIELD_RSSI,   PG_BLACKBOX_CONFIG, offsetof(blackboxConfig_t, fields) },
+    { "bb_log_gyro",                VAR_UINT16 | MASTER_VALUE | MODE_BITSET, .config.bitpos = SELECT_FIELD_GYRO,   PG_BLACKBOX_CONFIG, offsetof(blackboxConfig_t, fields) },
+    { "bb_log_acc",                 VAR_UINT16 | MASTER_VALUE | MODE_BITSET, .config.bitpos = SELECT_FIELD_ACC,   PG_BLACKBOX_CONFIG, offsetof(blackboxConfig_t, fields) },
+    { "bb_log_debug",               VAR_UINT16 | MASTER_VALUE | MODE_BITSET, .config.bitpos = SELECT_FIELD_DEBUG,   PG_BLACKBOX_CONFIG, offsetof(blackboxConfig_t, fields) },
+    { "bb_log_motors",              VAR_UINT16 | MASTER_VALUE | MODE_BITSET, .config.bitpos = SELECT_FIELD_MOTOR,   PG_BLACKBOX_CONFIG, offsetof(blackboxConfig_t, fields) },
+    { "bb_log_gps",                 VAR_UINT16 | MASTER_VALUE | MODE_BITSET, .config.bitpos = SELECT_FIELD_GPS,   PG_BLACKBOX_CONFIG, offsetof(blackboxConfig_t, fields) },
```

Comment:
Should be named `blackbox_`, not `bb_`. Beginning users won't know what "bb" means. Also change to `VAR_UINT32` after changing the type in the parameter group.